### PR TITLE
Copied over release note from PAB 10.17 release notes (PAB-3608)

### PR DESCRIPTION
**Pull request by kailol-sag**

### CODE REVIEW

**Review by BeateRixen — APPROVED** [review]

File: content/release-10-17-0/announcements-10-17-0.md [hunk 1]
```diff
@@ -42,6 +42,26 @@ declaration for applications to start. Refer to the [Spring Security documentati
...
+However, it is unlikely that a model should be attempting to clear the same alarm multiple times,
+and the previous behavior had the effect of sending numerous unnecessary HTTP requests, potentially causing a drop in performance.
+For more information on the `pulse` type and how inputs are converted, see
+[The pulse type](https://documentation.softwareag.com/pab/10.17.0/en/webhelp/pab-webhelp/index.html#page/pab-webhelp%2Fco-AnaBui_pulse_type.html) and
+[Type conversions](https://documentation.softwareag.com/pab/10.17.0/en/webhelp/pab-webhelp/index.html#page/pab-webhelp%2Fco-AnaBui_type_conversions.html),
+both in the Analytics Builder documentation.
+
+##### Changes to microservice health endpoint
+
+For reasons of security and performance, the REST endpoint `/service/cep/health` no longer returns a comprehensive list of status values. All of the same information is still available from REST endpoints under `/service/cep/diagnostics/...`.
```
Comment (skom-sag) [inline content/release-10-17-0/announcements-10-17-0.md:63]: can we move this to a separate PR?

There was an issue with this change and it got reverted, Richard will be making another attempt to get this in but it would be good to isolate this for now

Comment (kailol-sag) [reply content/release-10-17-0/announcements-10-17-0.md:63]: oh, I did not add this. I was only moving this to a different location.
Can we keep it for now? And then, when @rthrippleton cannot get it in, we can remove it on a separate PR?

**Comment by rthrippleton** [comment]
Whatever happens I'll get the health endpoint change in this release, even if I can't get it working for readiness (the problem we're having), so don't worry.

________________________________

File: content/release-10-17-0/announcements-10-17-0.md (hunk 1 above)
Comment (rthrippleton) [reply content/release-10-17-0/announcements-10-17-0.md:63]: Whatever happens I'll get the health endpoint change in this release, even if I can't get it working for readiness (the problem we're having), so don't worry.

**Review by skom-sag — APPROVED** [review]
